src/intl/en/learn-quizzes.json

```diff
@@ -326,6 +326,51 @@
+  "i002-c-label": "Ventures and grants, pool capital and vote on projects to fund",
+  "i002-c-explanation": "Venture or grants DAOs are one example, but DAOs are not limited to that.",
+  "i002-d-label": "All of the above",
+  "i002-d-explanation": "A DAO can have a multitude of 'missions'. Check out the resources to find practical examples.",
+  "i003-prompt": "Unlike traditional organizations, DAOs are…",
+  "i003-a-label": "Usually hierarchical",
+  "i003-a-explanation": "DAOs are usually flat, and fully democratized.",
+  "i003-b-label": "Transparent and fully public about their activities",
+  "i003-b-explanation": "Thanks to on-chain voting, decisions are transparent on the blockchain. Discussions and other elements of the decision-making process are usually open as well.",
+  "i003-c-label": "Controlled by a central party",
+  "i003-c-explanation": "Changes require voting by the members. Services offered are handled automatically in a decentralized manner.",
+  "i003-d-label": "Restricted regarding who can suggest changes",
+  "i003-d-explanation": "Usually, every DAO member can suggest changes.",
+  "i004-prompt": "What is essential about Smart Contracts for DAOs?",
+  "i004-a-label": "The Smart Contract code can be modified",
+  "i004-a-explanation": "Once the contract is live on Ethereum, no one can change the rules except by a vote. This allows the DAO to run by the rules it was programmed with.",
+  "i004-b-label": "One DAO member gets the right to spend money from the treasury",
+  "i004-b-explanation": "The treasury is defined by the smart contract. To spend money, the group's approval is needed.",
+  "i004-c-label": "Trust in the underlying blockchain due to a distributed consensus",
+  "i004-c-explanation": "It’s important for a DAO that the underlying blockchain cannot be manipulated. Ethereum’s own consensus is distributed and established enough for organizations to trust the network.",
+  "i004-d-label": "DAOs don’t need Smart Contracts",
+  "i004-d-explanation": "The backbone of a DAO is its smart contract, which defines the rules of the organization and holds the group's treasury.",
+  "i005-prompt": "What is not a mechanism to govern a DAO?",
+  "i005-a-label": "Token-based membership",
+  "i005-a-explanation": "Token-based governance is very widely used. It’s usually fully permissionless and is typically used to govern broad decentralized protocols and/or tokens themselves.",
+  "i005-b-label": "Share-based membership",
+  "i005-b-explanation": "Share-based DAOs are more permissioned but still quite open. Any prospective member can submit a proposal to join the DAO, usually offering a tribute of some value in the form of tokens or work.",
+  "i005-c-label": "Reputation-based membership",
+  "i005-c-explanation": "Unlike token or share-based membership, reputation-based DAOs don't transfer ownership to contributors. DAO members must earn reputation through participation.",
+  "i005-d-label": "Residency-based membership",
```

Comment:
Might push back here... this isn't listed on the page as a type of DAO, but there are definitely (at the very least experimental) DAOs based on various types of residency. Could we find another false example here?

Reply:
I thought about a better example, but didn't really come up with one... theoretically, there's a lot of possible critera. 
I know a couple "local" DAOs, but wouldn't think that any of them cares about nationality/residency. You're welcome to adapt this.